src/main/java/org/molgenis/vcf/inheritance/genemapper/GenemapConverter.java

```diff
@@ -0,0 +1,82 @@
+package org.molgenis.vcf.inheritance.genemapper;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Set;
+import org.molgenis.vcf.inheritance.genemapper.model.GeneInheritanceValue;
+import org.molgenis.vcf.inheritance.genemapper.model.InheritanceMode;
+import org.molgenis.vcf.inheritance.genemapper.model.Phenotype;
+
+public class GenemapConverter {
+
+  private GenemapConverter() {}
+
+  public static final String COMMENT_PREFIX = "#";
+
+  public static void run(Path input, Path ouput) {
+    try {
+      List<String> omimlines = Files.readAllLines(input);
+      BufferedWriter writer = new BufferedWriter(new FileWriter(ouput.toFile()));
+      omimlines.stream()
+          .filter(line -> !line.startsWith(COMMENT_PREFIX))
+          .map(OmimLineMapper::parseOmimLine)
+          .forEach(omimHits -> writeToFile(writer, omimHits));
+      writer.close();
```

Comment:
stream not closed when previous statement throws an exception. use a try-with-resources statement.